src/renderers/shared/fiber/ReactFiberCommitWork.js

```diff
@@ -321,22 +323,25 @@ module.exports = function<T, P, I, TI, C>(
         }
         if (finishedWork.effectTag & Callback) {
           if (finishedWork.callbackList) {
-            callCallbacks(finishedWork.callbackList, instance);
+            const callbackErrors = callCallbacks(finishedWork.callbackList, instance);
```

Comment:
So following up from my previous comment, this should return either a single error or null